server/controllers/userSignUp.js

```diff
@@ -0,0 +1,63 @@
+import express from 'express';
+import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+const models = require('../models');
+const signupValidation = require('../validation/userSignup');
+
+/** /register:
+   post:
+      description: client can register
+      responses: "200"
+       description: A successful registration
+*/
+const signUp = async (req, res) => {
+  try {
+    const { error } = signupValidation(req.body);
+    if (error) {
+      return res.status(400).send(error.details[0].message);
+    }
+    const clientData = await models.Clients.create({
+      firstName: req.body.firstName,
+      lastName: req.body.lastName,
+      email: req.body.email,
+      password: bcrypt.hashSync(req.body.password, 10),
+      phoneNumber: req.body.phoneNumber,
+      address: req.body.address,
+      isAdmin: req.body.isAdmin,
+    });
+    await models.Clients.findOne({
+      where: {
+        email: req.body.email,
+      },
+    });
+    if (!clientData) {
+      res.send({
+        message: 'client not found',
+      });
```

Comment:
why are you checking on the client again

Reply:
I will remove it then